scripts/katana_version/__main__.py

```diff
@@ -569,6 +569,21 @@ def setup_update_dependent_pr_subcommand(subparsers):
     parser.set_defaults(subcommand_impl=update_dependent_pr_subcommand)
 
 
+def tag_stable_subcommand(args):
+    config: Configuration = args.configuration
+    ke_commit = args.commit
+
+    check_remotes(config)
+    check_clean(args, config)
+
+    version = get_version(config, commit=ke_commit)
+    tag = f"stable-{version}"
+
+    git.tag_commit(tag, ke_commit, config.enterprise)
+    k_commit = git.submodule_commit_at(SUBMODULE_PATH, ke_commit, config.enterprise)
+    git.tag_commit(tag, k_commit, config.open)
```

Comment:
Doesn't this need to push the tags to the remote for them to actually appear on github? The other commands push their commits upstream, so it makes sense to push the tags at least if `really` is set.